graph/api/src/main/scala/org/apache/spark/graph/api/CypherSession.scala

```diff
@@ -50,14 +52,14 @@ object CypherSession {
   val LABEL_COLUMN_PREFIX = ":"
 
   /**
-   * Extracts [[NodeFrame]]s from a [[Dataset]] using column name conventions.
+   * Extracts [[NodeDataset]]s from a [[Dataset]] using column name conventions.
    *
    * For information about naming conventions, see [[CypherSession.createGraph]].
    *
    * @param nodes node dataset
    * @since 3.0.0
    */
-  def extractNodeFrames(nodes: Dataset[Row]): Set[NodeFrame] = {
+  def extractNodeDataset(nodes: Dataset[Row]): Set[NodeDataset] = {
```

Comment:
Since it produces a `Set[]`, do you think a plural form may work better?
```suggestion
  def extractNodeDatasets(nodes: Dataset[Row]): Set[NodeDataset] = {
```